pykafka/balancedconsumer.py

```diff
@@ -165,6 +172,7 @@ def __init__(self,
         self._zookeeper_connect = zookeeper_connect
         self._zookeeper_connection_timeout_ms = zookeeper_connection_timeout_ms
         self._reset_offset_on_start = reset_offset_on_start
+        self._use_rdkafka = rdkafka and use_rdkafka
```

Comment:
To avoid confusion, I think this should raise an exception if `rdkafka == False and use_rdkafka == True`.